modules/mauth-authenticator-akka-http/src/test/scala/com/mdsol/mauth/akka/http/MAuthDirectivesSpec.scala

```diff
@@ -184,4 +196,229 @@ class MAuthDirectivesSpec extends WordSpec with Matchers with ScalatestRouteTest
       }
     }
   }
+
+  "authenticate for V2" should {
```

Comment:
Should we add test cases for the following?

* header name detection should be case insensitive
* The default RequestAuthenticator (where `v2OnlyAuthenticate = false`) will extract/handle both v1 and v2 headers, with v2 headers taking precedence. This should be under the `"authenticate"` test section

Reply:
added and updated as suggested https://github.com/mdsol/mauth-jvm-clients/pull/92/commits/735bd0daeb6afd8ec05d1562413a7721f033797f